src/main/java/com/example/board/domain/hobby/repository/HobbyRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.board.domain.hobby.repository;
+
+import com.example.board.domain.hobby.entity.Hobby;
+import com.example.board.domain.hobby.entity.HobbyType;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface HobbyRepository extends JpaRepository<Hobby, Long> {
+
+    @Query("select h from Hobby h where h.hobbyType = :hobbyType")
+    Hobby findByHobbyType(HobbyType hobbyType);
```

Comment:
```suggestion
    Hobby findByHobbyType(HobbyType hobbyType);
```

위와 같이 Spring Data JPA의 쿼리 메서드를 사용하지 않고 JPQL을 작성하여 사용하신 이유가 있으신가요?

Reply:
쿼리 메서드를 사용하지 않은 이유는
`네이밍이 살짝만 달라져도 제대로 동작하지 않으니 JPQL이 저에게는 더 확실하다고 느껴져서 이용했습니다.`

하지만 실제로
현재 제가 작성한 메서드는 단순한 조회이기도하고 
때문에 쿼리 메서드를 사용하는 편이 더 생산성이 높을거 같다고 생각이 들었습니다.

아직 쿼리 메서드를 이용하는데 제가 익숙하지 않아서 그랬던거 같습니다.
`지금과 같이 쿼리 메서드의 이점을 확실하게 가져올 수 있는 부분에서는 이용해보는 편이 좋다고 생각이 들어 상황에 따라 맞는 방식을 더 생각해보겠습니다 !`